acme/setup.py

```diff
@@ -4,7 +4,7 @@
 from setuptools import find_packages
 
 
-version = '0.2.0.dev0'
+version = '0.2.0dev0'
```

Comment:
why do you remove the dot? `version` has been kept so far in canonical form, with dot